tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmSubscriptionManager.cs

```diff
@@ -3,23 +3,46 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
+
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Numerics;
 using Datadog.Trace.Logging;
+using Datadog.Trace.RemoteConfigurationManagement.Protocol;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Serilog.Events;
 
 namespace Datadog.Trace.RemoteConfigurationManagement;
 
 internal class RcmSubscriptionManager : IRcmSubscriptionManager
 {
+    private const int RootVersion = 1;
+
     public static readonly IRcmSubscriptionManager Instance = new RcmSubscriptionManager();
     private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<RcmSubscriptionManager>();
 
     private readonly List<ISubscription> _subscriptions = new();
     private readonly object _syncRoot = new();
+
+    /// <summary>
+    /// Key is the path
+    /// </summary>
+    private readonly Dictionary<string, RemoteConfigurationCache> _appliedConfigurations = new();
+
+    private readonly string _id;
+
+    private string? _backendClientState;
+    private int _targetsVersion;
     private BigInteger _capabilities;
 
+    public RcmSubscriptionManager()
+    {
+        DatadogLogging.GetLoggerFor<RcmSubscriptionManager>().Debug("New RcmSubscriptionManager");
```

Comment:
I think we already have one logger instance here?

```suggestion
        Log.Debug("New RcmSubscriptionManager");
```

Reply:
Right, that's debug logging I forgot to remove 🤦